docs/celsius-converter-exception/index.md

```diff
@@ -4,26 +4,7 @@ Our second widget will be one that takes a temperature value in Celsius and
 converts it to Fahrenheit. Create a new file in `src` called
 `CelsiusConverter.re`:
 
-```reason
-let convert = celsius => 9.0 /. 5.0 *. celsius +. 32.0;
-
-[@react.component]
-let make = () => {
-  let (celsius, setCelsius) = React.useState(() => "");
-
-  <div>
-    <input
-      value=celsius
-      onChange={evt => {
-        let newCelsius = ReactEvent.Form.target(evt)##value;
-        setCelsius(_ => newCelsius);
-      }}
-    />
-    {React.string({js|°C = |js})}
-    {React.string({js|? °F|js})}
-  </div>;
-};
-```
+<<< Snippets.re#celsius-converter-v1
```

Comment:
I guess a small disadvantage is that when you edit these docs, you have to keep the dev server open in a tab so you can see what it renders

Reply:
True, but it doesn't looks like a big deal, as looking at the document rendered by the server is generally the "final proof" to make sure everything is ok. Similar thing happens with melange.re and mkdocs.